Tests/check_wheel.py

```diff
@@ -0,0 +1,38 @@
+import importlib.util
+import sys
+
+from PIL import features
+
+
+def test_wheel_modules():
+    expected_modules = {"pil", "tkinter", "freetype2", "littlecms2", "webp"}
+
+    # tkinter is not available in cibuildwheel installed CPython on Windows
+    if not importlib.util.find_spec("tkinter"):
```

Comment:
I expect there's a reason, but why is it not simpler to test for tkinter by attempting to import it normally?

Reply:
Lint error, https://github.com/nulano/Pillow/actions/runs/6998411214/job/19036420674#step:7:21:
```
Tests/check_wheel.py:10:16: F401 `tkinter` imported but unused; consider using `importlib.util.find_spec` to test for availability
```

Reply:
Ah, `assert tkinter` should silence the warning. We've done that before.

Reply:
Yes, I remember seeing that before, but figured the linter suggestion would work as well.

I've pushed a commit to `try: import tkinter` instead for consistency with other optional imports (the only one I could find with a simple grep was in `Image.preinit`).

Reply:
Yeah, the others were replaced with `pytest.importorskip` (https://github.com/python-pillow/Pillow/pull/4436) which doesn't fit here.